apps/vault/contracts/Vault.sol

```diff
@@ -104,6 +106,17 @@ contract Vault is VaultBase {
         return supportedInterfaceDetectionERCs[interfaceDetectionERC];
     }
 
+    // Proxy implementation
+    function proxyType() public pure returns (uint256 proxyTypeId) {
+        return UPGRADEABLE;
```

Comment:
Anyway, for this to work we need to wait for this https://github.com/aragon/aragonOS/pull/281 to be merged and then update aragonOS dependency.

Reply:
About the type, I see your point, but I guess I would then change the ERC897 spec to add this new type. Otherwise we would be here using a standard and breaking it. Reading again the [definition](https://github.com/ethereum/EIPs/pull/897):
"Upgradeable proxy (id = 2): The proxy code address can be changed depending
on some arbitrary logic implemented either at the proxy level or in its forwarded
logic."
I think the problem is the name which is misleading, because I think this Vault fits in this definition as a proxy.